server/ban_manager.py

```diff
@@ -32,10 +34,21 @@ def load_banlist(self):
         except FileNotFoundError:
             return
 
+    def load_hdidbanlist(self):
+        try:
+            with open('storage/hdidbanlist.json', 'r') as hdidbanlist_file:
+                self.hdidbans = json.load(hdidbanlist_file)
```

Comment:
you need to handle `ValueError` as well if you don't want it to crash on JSON parse error